igvm/hypervisor_preferences.py

```diff
@@ -162,18 +211,27 @@ def __call__(self, vm, hv) -> bool:
         hv_cpu_threshold = float(self.hv_cpu_thresholds[hv_model])
         hv_cpu_util_overall = hv.estimate_cpu_usage(vm)
 
-        return (
-            hv_cpu_util_overall is not None and
-            hv_cpu_util_overall > hv_cpu_threshold
-        )
+        # if there is no value we assume it's a fresh hv
+        if hv_cpu_util_overall is None:
+            return 1.
+
+        # since this is a limiting preference, we don't want any vm end up on
+        # the hv that would exceed the cpu threshold
+        if hv_cpu_util_overall > hv_cpu_threshold:
+            return False
 
+        # normalize the value. this is only valid for "exhaution values" like
+        # cpu_util_pct and iops_avg
+        return 1 - (hv_cpu_util_overall / hv_cpu_threshold)
```

Comment:
If there is a slight chance that this becomes a integer we should better cast it to float.

Reply:
done